app/menu/template.hbs

```diff
@@ -0,0 +1,13 @@
+<h2>Menu</h2>
+
+{{#group-categories items=model class="menu" tagName='ul' as |category|}}
+  <li>
+    <h3>{{category.name}}</h3>
+    <ul>
+      {{#each category.items as |menuItem|}}
+        {{menu-item-result menuItem=menuItem}}
+        {{mdl-button class="add-btn" isRaised=true text='Add To Cart'}}
```

Comment:
The add to cart button should be within the template of the `menu-item-result` otherwise you'll get the strange layout that you have right now. You'll also want to add an `on-click` on the `mdl-button` which is an action to be triggered when the button is clicked, we'll look at this in the review lesson on actions with Ember.